xbmc/cores/VideoPlayer/DVDCodecs/Video/VTB.h

```diff
@@ -19,24 +19,48 @@
  */
 #pragma once
 
-#include "system_gl.h"
+#ifdef HAS_GL
+#include <OpenGL/gl.h>
+#else
+#include <OpenGLES/ES2/gl.h>
+#endif
```

Comment:
this change isn't enough, have a look at https://github.com/xbmc/xbmc/blob/f743b10/xbmc/system.h#L189-L193
system.h undefines `HAS_GL` for iOS